pkg/controllers/vdb/k8s.go

```diff
@@ -92,11 +92,29 @@ func createSts(ctx context.Context, vrec config.ReconcilerInterface, expSts *app
 	return vrec.GetClient().Create(ctx, expSts)
 }
 
-// createSts will create a new deployment. It assumes the deployment doesn't already exist.
+// createDep will create a new deployment. It assumes the deployment doesn't already exist.
 func createDep(ctx context.Context, vrec config.ReconcilerInterface, vpDep *appsv1.Deployment, vdb *vapi.VerticaDB) error {
 	err := ctrl.SetControllerReference(vdb, vpDep, vrec.GetClient().Scheme())
 	if err != nil {
 		return err
 	}
```

Comment:
Now the deployment belongs to the sts so you should remove this.

Reply:
Removed in https://github.com/vertica/vertica-kubernetes/pull/980/commits/2cb807d2a895bc50d4b73093cb4bd4e3f2a49028